cardano-cli/test/cardano-cli-golden/Test/Golden/Governance/DRep.hs

```diff
@@ -168,12 +168,14 @@ hprop_golden_governance_drep_retirement_certificate_id_bech32 =
     bracketSem drepRetirementCertSem $
       H.diffFileVsGoldenFile certFile
 
+-- Execute me with:
+-- @cabal test cardano-cli-test --test-options '-p "/governance drep metadata hash/"'@
 hprop_golden_governance_drep_metadata_hash :: Property
 hprop_golden_governance_drep_metadata_hash = propertyOnce . H.moduleWorkspace "tmp" $ \tempDir -> do
   goldenDRepMetadataHash <- H.note "test/cardano-cli-golden/files/golden/governance/drep/drep_metadata_hash"
-
-  drepMetadataFile <- noteTempFile tempDir "drep-metadata.json"
-  H.evalIO $ writeFile drepMetadataFile "{ \"Lorem\": \"ipsum\", \"dolor\": \"sit\", \"amet\": \"consectetur\" }"
```

Comment:
Can you keep this existing test (i.e. make yours an additional one)? So that we check that the existing hash isn't changed by this PR.